packages/components/src/Components/ConditionalFilter/Group.js

```diff
@@ -285,11 +285,12 @@ class Group extends Component {
                             className="pf-c-select__menu-item"
                             variant={showMore.variant}
                             onClick={showMore.items[0].onClick}
+                            value="Show more"
                         >
                             {showMore.items[0].label}
                         </Button>
                     </SelectGroup>
-                    : <Fragment />}
+                    : <Fragment value="" />}
```

Comment:
When we are here we can do this

```jsx
onShowMore && <SelectGroup>
                        <Button
                            {...showMore.items[0]}
                            className="pf-c-select__menu-item"
                            variant={showMore.variant}
                            onClick={showMore.items[0].onClick}
                            value="Show more"
                        >
                            {showMore.items[0].label}
                        </Button>
                    </SelectGroup>
```

to make it less ugly. :smile:

Reply:
Not really, because PF complains that `React.children([false]).map(child => child.props.value.toString() === 'something')` (`false` is calculated from this expression) is not working :disappointed:

Reply:
:boom: